packages/utils/fetch_error_schema.js

```diff
@@ -2,8 +2,8 @@ const https = require('https');
 const fs = require('fs');
 
 const ERROR_SCHEMA_URL =
-    'https://raw.githubusercontent.com/near/nearcore/master/chain/jsonrpc/res/rpc_errors_schema.json';
-const TARGET_SCHEMA_FILE_PATH = `${process.argv[2] ||  process.cwd()}/src/errors/rpc_error_schema.json`;
+    'https://raw.githubusercontent.com/nearprotocol/nearcore/4c1149974ccf899dbcb2253a3e27cbab86dc47be/chain/jsonrpc/res/rpc_errors_schema.json';
+const TARGET_SCHEMA_FILE_PATH = `${process.argv[2] ||  process.cwd()}/src/errors/rpc_error_schema.json'`;
```

Comment:
Not sure if we should use a rpc schema from 2020

Reply:
the newest version was causing a few tests to crush, but I managed to fix them